frontend/src/pages/modelRegistry/screens/RegisterModel.tsx

```diff
@@ -0,0 +1,319 @@
+            modelLocationURI,
+        },
+        setData,
+        resetData,
+    ] = useRegistrModelData();
+    const [loading, setIsLoading] = React.useState(false);
+    const [error, setError] = React.useState<Error | undefined>(undefined);
+    const { modelRegistry: mrName } = useParams();
+
+    React.useEffect(() => {
+        setData('modelRegistryName', mrName || "");
+    }, [setData]);
+
+    const isDisabled =
+        !modelRegistryName ||
+        !modelName ||
+        !versionName ||
+        loading ||
+        (modelLocationType === 'URI' && !modelLocationURI) ||
+        (modelLocationType === 'Object storage' &&
+            (!modelLocationBucket || !modelLocationEndpoint || !modelLocationPath));
+
+    const handleSubmit = () => {
+        setIsLoading(true);
+        setError(undefined);
+        /* eslint-disable-next-line no-console */
+        console.log({
+            modelRegistryName,
+            modelName,
+            modelDescription,
+            versionName,
+            versionDescription,
+            sourceModelFormat,
+            modelLocationType,
+            modelLocationEndpoint,
+            modelLocationBucket,
+            modelLocationRegion,
+            modelLocationPath,
+            modelLocationURI,
+        });
+    };
+
+    return (
+        <ApplicationsPage
+            title="Register model"
+            description="Create a new model and register a first version of the new model"
+            breadcrumb={
+                <Breadcrumb>
+                    <BreadcrumbItem render={() => <Link to="/modelRegistry">Model registry</Link>} />
+                    <BreadcrumbItem render={() => <Link to={`/modelRegistry/${modelRegistryName}`}>{modelRegistryName}</Link>} />
+                    <BreadcrumbItem>Register model</BreadcrumbItem>
+                </Breadcrumb>
+            }
+            loaded
+            empty={false}
+            provideChildrenPadding
+        >
+            <Form>
+                <Stack hasGutter>
+                    <StackItem>
+                        <FormGroup label="Model registry" isRequired fieldId="mr-name">
+                            <TextInput
+                                isDisabled
+                                isRequired
+                                type="text"
+                                id="mr-name"
+                                name="mr-name"
+                                value={modelRegistryName}
+                            />
+                        </FormGroup>
+                    </StackItem>
+                    <StackItem>
+                        <FormSection
+                            title={
+                                <>
+                                    <Title headingLevel="h2">Model info</Title>
+                                    <Text component="p" className="form-subtitle-text">
+                                        Configure the model info you want to create
```

Comment:
In the mockups, each of these subtitles has a period at the end of the sentence. We should probably do that (although the description doesn't -- maybe let's look at what other forms do and be consistent)

Reply:
![image](https://github.com/user-attachments/assets/cc8f7708-03ec-46d7-83d9-02cf8b0d7088)
No period here in mock, but i can add.